### ISSUE TITLE
<ion-input> with *ngIf breaks tab focus order

### ISSUE THREAD

**Issue by imgx64** [ionic-team/ionic#9329]
**Ionic version:**
[x] **2.x**

**I'm submitting a ...**
[x] bug report

**Current behavior:**
If you have an `<ion-input>` that gets added/removed using `*ngIf`, its tab focus order becomes the last, even if it's in the middle.

**Expected behavior:**
It should have the correct tab focus order.

**Steps to reproduce:**
1. Create a new Ionic v2 project
2. Paste the below code in `src/pages/home/home.html`.
3. Run `ionic serve`
4. Click on `Show Bar` to check it
5. Click on `Foo`
6. Press <kbd>Tab</kbd>
7. Notice that `Baz` is focused instead of `Bar`

```
<ion-content padding>
  <ion-list>
    <ion-item>
      <ion-label>Show Bar</ion-label>
      <ion-checkbox [(ngModel)]="showBar"></ion-checkbox>
    </ion-item>

    <ion-item>
      <ion-label floating>Foo</ion-label>
      <ion-input type="text"></ion-input>
    </ion-item>

    <ion-item *ngIf="showBar">
      <ion-label floating>Bar</ion-label>
      <ion-input type="text"></ion-input>
    </ion-item>

    <ion-item>
      <ion-label floating>Baz</ion-label>
      <ion-input type="text"></ion-input>
    </ion-item>
  </ion-list>
</ion-content>
```

For some reason, I can't reproduce it on Plunker.

**Ionic info:**
```
Your system information:

 ordova CLI: 6.4.0
Ionic CLI Version: 2.1.12
Ionic App Lib Version: 2.1.7
ios-deploy version: Not installed
ios-sim version: Not installed
OS: Windows 10
Node Version: v4.6.1
Xcode version: Not installed
```
